index.html

```diff
@@ -1,132 +1,89 @@
-                    </p>
-                    <p>
-                        Hi, I'm Phillip Whisenhunt(<em>wis-n-hunt</em>). I'm currently a 100% remote Full Stack Software Engineer at <a href="https://www.olark.com" target="_blank">Olark</a> where I work on everything from building Python services for Olark's Service Oriented Architecture to building React and Redux applications powered by GraphQL.
-                    </p>
-                    <p>
-                        In a past life I purused a PhD and career in academia, which afforded me the opportunity to work and publish in a variety of research areas such as biometrics, bioinformatics, and mobile device security.
-                    </p>
-                </div>
-                <div class="col-md-2"></div>
-            </div>
-            <div class="row">
-                <div class="col-md-2"></div>
-                <div class="col-md-8">
-                    <h4>Writing</h4>
-                    <article>
-                      <a href="https://blog.olark.com/how-a-saas-company-evolves-its-front-end-tech-stack" alt="How a SaaS company evolves its front-end tech 
-">Olark Blog: How a SaaS company evolves its front-end tech stack</a></h3>
-                        <p>
-                          Article describing how Olark has continually evolved it's front-end technology stack.
-                        </p>
-                    </article>
-                    <article>
-                      <a href="http://www.smashingmagazine.com/2013/08/09/backbone-js-tips-patterns/" alt="Backbone.js Tips and Patterns">Smashing Magazine: Backbone.js Tips and Patterns</a>
-                        <p>
-                          Article detailing various tips and patterns for the popular JavaScript framework Backbone.js.
-                        </p>
-                    </article>
-                    <article>
-                      <a href="http://www.manning.com/breed/" alt="Backbone.js in Action">Backbone.js in Action</a>
-                        <p>
-                          I wrote the Modular Javascript chapter for Backbone.js in Action.
-                        </p>
-                    </article>
-                    <h4>Contact</h4>
-                    <p>
-                        By far the best way to contact me is through email at phillip dot whisenhunt at gmail dot com. Take care.
-                    </p>
-                </div>
-                <div class="col-md-2"></div>
+<main class="home-main-content">
+    <section class="page-heading-section">
+        <h1>
+            Hello.<br/>
+            I'm <span class="heading-emphasis">Phillip Whisenhunt.</span><br/>
+            I'm a Full Stack Software Engineer.<br/>
+        </h1>
+    </section>
+    <section id="personal-information-section" class="personal-information-section">
+        <div class="information-container">
```

Comment:
Maybe use a more descriptive name then information but then again naming is hard 😛